ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Table/ScanConfigsTablePage.tsx

```diff
@@ -243,7 +244,7 @@ function ScanConfigsTablePage({
             {error ? (
                 <PageSection variant="light" isFilled id="policies-table-error">
                     <Bullseye>
-                        <Alert variant="danger" title={error} />
+                        <Alert variant="danger" title={getAxiosErrorMessage(error)} />
```

Comment:
@bradr5 the type of `title` here is `ReactNode`, which doesn't like the `error` object passed in after the upgrade. Will this change do what is expected here?